hnn_core/network_models.py

```diff
@@ -249,3 +255,90 @@ def law_2021_model():
         src_cell, target_cell, loc, receptor, weight, delay, lamtha)
 
     return net
+
+
+# Remove params argument after updating examples
+# (only relevant for Jones 2009 model)
+def _calcium_model(params, add_drives_from_params):
+    """Instantiate the Jones 2009 model with improved calcium dynamics.
+
+    Returns
+    -------
+    net : Instance of Network object
+        Network object used to store the Jones 2009 model with an impoved
+        calcium channel distribution.
+    Notes
+    -----
+    This model builds on the Jones 2009 model by using a more biologically
+    accurate distribution of calcium channels on L5 pyramidal cells.
+    Specifically, this model introduces a distance dependent maximum
+    conductance (gbar) on calcium channels such that the gbar linearly
+    decreases along the apical dendrite in the direction of the soma.
+    """
+    hnn_core_root = op.dirname(hnn_core.__file__)
+    params_fname = op.join(hnn_core_root, 'param', 'default.json')
+    if params is None:
+        params = read_params(params_fname)
+
+    net = jones_2009_model(params, add_drives_from_params)
+
+    # Replace L5 pyramidal cell template with updated calcium
+    cell_name = 'L5_pyramidal'
+    pos = net.cell_types[cell_name].pos
+    net.cell_types[cell_name] = pyramidal_ca(
+        cell_name=_short_name(cell_name), pos=pos)
+
+    return net
+
+
+def add_default_ERP(net, tstart=0.0):
```

Comment:
but this is hardcoding all the parameters. Do you not foresee having to change a few of them? Maybe at leas the timing of the drives?

Reply:
```suggestion
def add_default_erp(net, tstart=0.0):
```

function name should be all lowercase

Reply:
@cjayb what's your feeling about this as compared to `add_drives_from_params` function? Are they both needed?

Reply:
> but this is hardcoding all the parameters.

This is a really good concern, but so are the default drives from the `.param`. Maybe the documentation should be clearer, but I was thinking that this is something that is exclusively tuned to the `default_network.py`, and we're finding frequently in the examples that we just want to copy and paste those exact parameters.

My feeling is that we either have to commit to only using the drives API for creating these boilerplate oscillations, or distribute the "representative" drives that are tuned to the model

Reply:
And yes I do think they exact parameters will be updated, but in a similar sense to `default_network`. It's distributing a reliable simulation that we have validated ourselves, and reflects our own state-of-the-art.

Reply:
I'm a huge fan of getting rid of `add_drives_from_params`! @jasmainak asked me once why I don't use the new API: in our examples it amounts to boilerplate and detracts from the main message.

I really like your proposal @ntolley, but could we name the function more explicitly?, e.g.,

```python
add_erp_drives_to_default_network(net)
```

And then include a passage in the howto/tutorial stating something like: "The module ``network_models.py`` contains functions that add specific sets of drives to specific networks. We recommend that users also modularise their code in this way, as it acts as documentation and eases reproducibility."

Then, in the docstring of each function, duplicate some of the tutorial text explaining why the drives are specified as they are, which figure of which paper they reproduce, etc.

Reply:
okay, so if the plan is to get rid of `add_drives_from_params`, could we make a separate PR that does that while introducing `add_erp_drives_to_default_network` ? I am afraid that having similar functions will confuse users and also future developers. And you know what's coming next ...

```sh
$ python -c 'import this' | sed -n 15p
There should be one-- and preferably only one --obvious way to do it.
```

Reply:
The issue with saving this function for another PR is we need to change the `default.json` param file to still use the `add_drives_from_params` functionality, otherwise the new parameter values won't get updated.

I can make it so `add_drives_from_params` actually just calls `add_erp_drives_to_default_network(net)` internally until we get rid of it. Either way I'd really like to avoid adding a bunch of uncessary code to the examples that utilize `add_drives_from_params`